<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>358</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>128</width>
          <height>384</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>134</width>
           <height>464</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>72</width>
             <height>72</height>
            </rect>
           </property>
           <attribute name="label">
            <string>家人</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>72</width>
             <height>72</height>
            </rect>
           </property>
           <attribute name="label">
            <string>黑名单</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>谷歌</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>百度</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string> bilibili</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnScrollArea">
         <property name="text">
          <string>Scroll Area</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToolBox">
         <property name="text">
          <string>Tool Box</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTabWidget">
         <property name="text">
          <string>Tab Widget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectCat">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>选中猫</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>284</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>104</width>
     <height>74</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>104</width>
     <height>74</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>156</width>
     <height>132</height>
    </rect>
   </property>
   <property name="title">
    <string>(Date/Time) Edit</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGif">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
